tools/setup-buildtools.cmd

```diff
@@ -1,13 +1,13 @@
 @echo off
-set "PATH=%ProgramFiles%\CMake\bin;%~dp0;%~dp0\vcpkg;%PATH%"
+set "PATH=%ProgramFiles%\CMake\bin;%~dp0;%~dp0vcpkg;%PATH%"
```

Comment:
So `%~dp0\vcpkg` would actually expand to (say) `C:\<path-to-this-batch-file>\\vcpkg`, which will still work correctly right? Asking because it looks `%~dp0\vcpkg` is more readable than `%~dp0vcpkg` in code :)

Reply:
Yes, seems `\\` also works here. I got some issue like this for string concatenation, so I'd like to make the tailing `\\` explicitly. Let me know if there is preference to add it back.